tests/acceptance/00_basics/macros/if_ignore_with_addsign.cf

```diff
@@ -0,0 +1,30 @@
+######################################################
+#
+#  Test that @if works for greater versions
```

Comment:
I guess this was copy/pasted. Should be `Test that @if also ignores lines starting with '@'`.